firmware/gpio/gpio.cpp

```diff
@@ -0,0 +1,157 @@
+/**************************
```

Comment:
Please create a header file. So that we can all first agree on the header first.

Reply:
Created a header file